### ISSUE TITLE
cellphone-phone and cellphone-android replaced with tray-arrow-down and tray-arrow-up

### ISSUE THREAD

**Issue by pjmdesi** [Templarian/MaterialDesign-Font#5]
For some reason, the glyphs for tray-arrow-up and tray-arrow-down are in the place of the mdi-cellphone-iphone and mdi-cellphone-android respectively.

on Cheatsheet:
mdi-cellphone-android -> F011D
mdi-cellphone-iphone -> F0120

In Font (screenshots of font open in Glyph mini app):
![(Friday 2021-08-13 @ 09 18 56) - Glyphs Mini - MaterialDesignIconsDesktop ttf - Material Design Icons Desktop](https://user-images.githubusercontent.com/36543053/129371428-f734db24-63bc-401f-9ddc-c7863215529b.png)
![(Friday 2021-08-13 @ 09 21 58) - Glyphs Mini - MaterialDesignIconsDesktop ttf - Material Design Icons Desktop](https://user-images.githubusercontent.com/36543053/129371664-2dc585b0-6a04-461c-be46-92c4760bbbb1.png)

In other words, copying the iPhone glyph from the cheatsheet, pasting into a document using the material design icons desktop font shows the tray arrow down glyph. I confirmed that I am using the most current version of the font file.

**Comment by pjmdesi** [comment]
Sorry, just realized that the HTML file in the repo has the updated glyph assignments. It seems the iphone and android version of the cellphone glyph have unfortunately been removed.

**Comment by Templarian** [comment]
@pjmdesi Please see the main repo's pinned issues we're working to get the 6.0 release out shortly. Part of that is removing and cleaning up legacy icons for the major release. With cellphones all looking the same we've decided to drop the brand specific designs.

https://github.com/Templarian/MaterialDesign
